vitrivr-engine-core/src/main/kotlin/org/vitrivr/engine/core/model/content/impl/cache/CachedAudioContent.kt

```diff
@@ -35,6 +36,7 @@ class CachedAudioContent(override val path: Path, override val channels: Short,
             }
             return buffer.asReadOnlyBuffer()
         }
+    override val id: UUID = UUID.randomUUID()
```

Comment:
I would add this to the constructor. There might be use-cases, where one wants to define this explicitly. You can also use default values in the constructor.